inference-engine/src/cldnn_engine/cldnn_config.cpp

```diff
@@ -26,6 +29,8 @@ using namespace InferenceEngine;
 
 namespace CLDNNPlugin {
 
+const int bit_shift = 2;
```

Comment:
what about more specifically...`#define CORE_TYPE_BIT_OFFSET 2` ?

Reply:
modified it.